apps/docs/content/components/Card/index.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Card
+status: Experimental
+source: https://github.com/primer/brand/tree/main/packages/react/src/Card/Card.tsx
+storybook: '/brand/storybook/?path=/story/components-card--default'
+description: Use Card to group information in a compact way and link to other pages.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+## Anatomy
+
+![An image showing the Card anatomy with an icon as a visual asset, a heading, a description and a CTA text in a vertical order.](https://github.com/primer/brand/assets/912236/633b83d8-4ea8-49bd-9d6c-45a1989060d9)
+
+### Visual asset
+
+The visual asset is optional and can be an image or an icon. It can be used to provide a visual cue to the user about the content of the card. To avoid visual noise, we recommend using only one icon or image, but not both in the same card.
```

Comment:
> but not both in the same card.

This would make a great Do/Dont illustration if you can do one 🙏. May go unnoticed otherwise.

Reply:
good point! 👍